src/components/KymaModules/KymaModulesCreate.js

```diff
@@ -349,6 +394,59 @@ export default function KymaModulesCreate({ resource, ...props }) {
         </MessageBox>,
         document.body,
       )}
+      {createPortal(
+        <MessageBox
```

Comment:
Not sure here, but we might move that `MessageBox` into a separate component. It might not be feasible if we need to pass a lot of props.